examples/interactive_asr/README.md

```diff
@@ -16,33 +16,48 @@ and the following models
 
 We recommend that you use [conda](https://docs.conda.io/en/latest/miniconda.html) to install the dependencies when available.
 ```bash
+# Assume that all commands are from the examples folder
+cd examples
+
 # Install dependencies
 conda install -c pytorch torchaudio
 conda install -c conda-forge librosa
 conda install pyaudio
```

Comment:
I don't see pyaudio installed in the scripts. I needed when I experimented with this. Is it needed?

Reply:
Yes this is correct, we need pyaudio. It is installed on this line
https://github.com/pytorch/audio/pull/248/files#diff-36b82146ca0e8499b5b86ff3882e063bR61

Reply:
Oh, right, thanks :)